azure-pipelines.yml

```diff
@@ -143,6 +143,10 @@ stages:
           parameters:
             pythonVersion: ${{ parameters.minimumPythonVersion }}
 
+        - template: ".azure/test-visual.yml"
+          parameters:
+            pythonVersion: ${{ parameters.minimumPythonVersion }}
+
```

Comment:
I think it would be better to leave this as a subcomponent of the `test-linux` run.  Our CI structure is deliberately designed such that the two-stage PR and one-stage push CI need a total of 10 VMs, which is our concurrency limit on Azure.  That lets us have 3 PRs working through their CI in parallel at peak throughput for release time - the stages are all roughly timed to take the same amount of wall time when run in parallel.

Reply:
moved the visual testing back to test-linux